roles/setup_replication/vars/PG_RedHat.yml

```diff
@@ -3,6 +3,7 @@ pg_data: "/var/lib/pgsql/{{ pg_version }}/{{ pg_instance_name }}/data"
 pg_default_data: "/var/lib/pgsql/{{ pg_version }}/{{ pg_instance_name }}/data"
 pg_log: "/var/log/postgres"
 pg_wal: ""
+pg_local_wal_archive_dir: ""
```

Comment:
Please define it in `defaults/main.yml`.

Reply:
Parameter removed and added to `defaults/main.yml`.